### ISSUE TITLE
Run MVD on Local Machine

### ISSUE THREAD

**Issue by zeier** [agera-edc/MinimumViableDataspace#191]
Feature #157
After #189, #56 

## Description

A deployment to Azure that uses Azure resources like KeyVault is provided by agera-edc/MinimumViableDataspace#139, which is a great way to get started with the MVD and deploy it to a cloud environment. 

To avoid the requirement to have an Azure subscription for hackathons and to conduct hackathons in a cloud agnostic way, it is required to run the MVD on a local machine.

_Note : In story #56 a docker-compose based approach using Azurite was created to run MVD for CI/CD without cloud dependencies. One pending component to add is the MVD UI (DataDashboard) in the docker-compose setup._

## Acceptance Criteria

- [x] The MVD can be run on a local machine based on step-by-step instructions 
- [x] Keys are stored on the local file system 
- [x] Documentation contains a security warning that keys will not be securely stored, and therefore only temporary keys for development purposes shall be used 
- [x] MVD can be executed locally on major platforms e.g. Windows, Linux based systems, MacOS
- [x] Integration test in CI to validate the local machine MVD

## Out of Scope
 
- Deployment to a local cluster, e.g. Minikube
- Additional data transfers in addition to blob storage

## Tasks

- [x] Add MVD UI in docker compose based setup created under #56.
- [x] Step by step documentation to run MVD locally.
- [x] Validate that MVD can be executed locally on major platforms e.g. Windows, Linux based systems, MacOS
- [x] Validate that the local machine MVD is validated in CI
